src/main/java/dev/openfeature/sdk/MutableStructure.java

```diff
@@ -0,0 +1,162 @@
+package dev.openfeature.sdk;
```

Comment:
I'm not following how/why this is a new file?

Reply:
`EvaluationContext` is an interface that extends `Structure`. `EvaluationContext` has a superset of `Structure`'s methods. This was the case previously, except it was done via encapsulation and proxying via lombok.

I've made them both interfaces now. `MutableStructure` is just a concrete implementation of `Structure`, and as before `MutableContext` encapsulates and proxies to `MutableStructure`. 

I hope that answers your question. This wasn't entirely necessary, but I think it's better than `Structure` remaining concrete, and duplicating it's methods in `EvaluationContext`, which is an interface and therefore couldn't extend it.